internal/config/config.go

```diff
@@ -244,6 +263,48 @@ func (c *Config) Keys(mount string) []string {
 	return nil
 }
 
+// migrateOptions is a best effort migration tool for when we introduce new options. It does not necessarily
+// handle worktree and env level options very well.
+func (c *Config) migrateOptions(migrations map[string]string) {
+	if nm := os.Getenv("GOPASS_CONFIG_NO_MIGRATE"); nm != "" {
+		return
+	}
+	var errs []error
+	debug.Log("migrateOptions running")
+	for oldK, newK := range migrations {
+		found := false
+		if val := c.root.GetGlobal(oldK); val != "" {
+			debug.Log("migrating option in root global store: %s -> %s ", oldK, newK)
+			errs = append(errs, c.root.SetGlobal(newK, val))
+			errs = append(errs, c.root.UnsetGlobal(oldK))
+			found = true
+		}
+		if val := c.root.GetLocal(oldK); val != "" {
+			debug.Log("migrating option in <root> local store: %s -> %s ", oldK, newK)
+			errs = append(errs, c.root.SetLocal(newK, val))
+			errs = append(errs, c.root.UnsetLocal(oldK))
+			found = true
+		}
+		for _, m := range c.Mounts() {
+			if cfg := c.cfgs[m]; cfg != nil {
+				if val := cfg.GetLocal(oldK); val != "" {
+					debug.Log("migrating option in local store %s: %s -> %s ", m, oldK, newK)
+					errs = append(errs, cfg.SetLocal(newK, val))
+					errs = append(errs, cfg.UnsetLocal(oldK))
+					found = true
+				}
+				if val := cfg.Get(oldK); !found && val != "" {
+					debug.Log("Found old option %s = %s in config, probably at the worktree or env level, "+
+						"or maybe at the system level cannot migrate it.", oldK, val)
+				}
+			}
+		}
+	}
+	if err := errors.Join(errors.Join(errs...)); err != nil {
```

Comment:
Why do you need to chain `errors.Join` twice?

Reply:
That's a mistake, I refactored that part a couple times and I messed up I guess 😅